modules/app.js

```diff
@@ -0,0 +1,38 @@
+import Store from './localStorage.js';
+
+import page from './script.js';
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+document.querySelector('.book-input').addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  if (title && author) {
+    const book = new Book(title, author);
+    page.bookList(book);
+    Store.addBook(book);
+  }
+
+  // to clear fields
+  e.target.reset();
+});
+
+// to remove a book
+document.querySelector('#book-collection').addEventListener('click', (e) => {
+  page.deleteBook(e.target);
+  Store.removeBook();
+});
+
+(function render() {
+  const books = Store.getBooks();
+  books.forEach((book) => {
+    page.bookList(book);
+  });
```

Comment:
[OPTIONAL] Kindly check your render method since it is not functioning as it's suppose to. Currently, you need to refresh the page before you can see the added books and a change in the time and date which is not in conjuction with best practices :-1: .

Reply:
Thank you